src/lib/stepper/stepper.md

```diff
@@ -156,6 +156,67 @@ by placing a `matStepperIcon` for each of the icons that you want to override. T
 
 Note that you aren't limited to using the `mat-icon` component when providing custom icons.
 
+#### Step States
+You can set the state of a step to whatever you want. The given state by default maps to an icon. However, it can be overridden the same way as mentioned above.
+
+```html
+<mat-horizontal-stepper>
```

Comment:
are you still planning to turn this into an embedded example?

Reply:
Oh yes sorry I did something else instead. I misunderstood the first time you mentioned it but now I understand. Will get on it.

Reply:
@mmalerba Not sure how to create an example for this because I would need to inject the `MAT_GLOBAL_STEPPER_OPTIONS` in the module and that would change the behavior of all of the steppers. Please advise. Thanks!

Reply:
Done!